public/new.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Noor</title>
+  </head>
+  <body>
+    <h1>NEW</h1>
+
+    <section class="new-item">
+      <img class="new-image" src="https://i.ytimg.com/vi/yzH8ul5PSZ8/maxresdefault.jpg" alt="multiple sclerosis">
+      <h2 class="new-title">Multiple sclerosis: Symptoms, causes, and treatment</h2>
+      <p class="new-description">Multiple sclerosis is a chronic disease that attacks the central nervous system. It affects the brain, spinal cord, and optic nerves.</p>
+      <button class="more">READ MORE</button>
+    </section>
+
+    <section class="new-item">
+      <iframe width="560" height="315" src="https://www.youtube.com/embed/zRzzDRZcUEA" frameborder="0" allowfullscreen></iframe>
```

Comment:
Are we going to embed the video on the new content page? Or will it just link to the video viewing page?